lib/quicly.c

```diff
@@ -3071,7 +3053,7 @@ int quicly_send(quicly_conn_t *conn, quicly_datagram_t **packets, size_t *num_pa
     } else if (conn->idle_timeout.at <= now) {
         LOG_CONNECTION_EVENT(conn, QUICLY_EVENT_TYPE_IDLE_TIMEOUT);
         conn->super.state = QUICLY_STATE_DRAINING;
-        destroy_all_streams(conn, 0);
+        destroy_all_streams(conn, 0, 0);
```

Comment:
```suggestion
        destroy_all_streams(conn, /*err=*/0, /*including_crypto_streams=*/0);
```
here, and any further calls to destroy_all_streams